File: 2/quadrado.py
#coding: utf-8
# 2) Crie uma classe que modele um quadrado e permita definir, alterar e consultar o tamanho dos lados e obter a área.
class Quadrado:

    def __init__(self, lado):
        self.lado = lado

    def obter_lado(self):
        return self.lado

    def alterar_lado(self, novo_lado):
        self.lado = novo_lado

    def area(self):
        return self.lado**2

File: 02/quadrado.rb
#coding: utf-8
#2) Crie uma classe que modele um quadrado e permita definir, alterar e consultar o tamanho dos lados e obter a área.
class Quadrado
	attr_accessor :lado

	def initialize(lado)
		@lado = lado
	end
	
	def area
		@lado*@lado
	end
end